### DISCUSSION TITLE
Add config RP2040 Lite Purple 16MB

### DISCUSSION THREAD

**Discussion by sav-lab** [micropython/micropython#11806] [RP2040 / Pico]
Hello! I created a config for the RP2040 Lite Purple 16MB board. The necessary files are in my repository:

https://github.com/sav-lab/LITE_PURPLE_16MB

I ask you to include support for this board in the main project.

Firmware files are:

https://github.com/sav-lab/LITE_PURPLE_16MB/tree/main/build-LITE_PURPLE_16MB

I tried them on the boards I bought, everything works.

**Comment by peterhinch** [comment]
Both links are 404.

**Comment by andypiper** [comment]
I think you've somehow managed to make the links you've pasted point somewhere invalid.

These seem to be what you meant?
- https://github.com/sav-lab/LITE_PURPLE_16MB
- https://github.com/sav-lab/LITE_PURPLE_16MB/tree/main/build-LITE_PURPLE_16MB

To get these included in the main repository I think you'd need to send a pull request for review.

**Comment by sav-lab** [comment]
fixed the links

**Reply by andypiper** [reply]
You will also need to send a Pull Request to the MicroPython project if you would like these considered for inclusion in the main project.
